docs/nodes/maintain/chain-config-flags.md

```diff
@@ -122,20 +122,47 @@ Specifies the directory for the Admin API to use to store CPU/Mem/Lock Profiles.
 
 Use the `eth-apis` field to specify the exact set of below services to enable on your node. If this field is not set, then the default list will be: `["eth","eth-filter","net","web3","internal-eth","internal-blockchain","internal-transaction"]`.
 
+:::note
+
+The names used in this configuration flag have been updated in coreth `v.0.8.14`.
+The previous names containing `public-` and `private-` are deprecated.
+While the current version continues to accept deprecated values, they may not be supported in future updates and updating to the new values is recommended.
+
+The mapping of deprecated values and their updated equivilant follows:
+
+| Deprecated                              | Use instead            |
+|-----------------------------------------|------------------------|
+| `public-eth`                            | `eth`                  |
+| `public-eth-filter`                     | `eth-filter`           |
+| `private-admin`                         | `admin`                |
+| `private-debug`                         | `debug`                |
+| `public-debug`                          | `debug`                |
+| `internal-public-eth`                   | `internal-eth`         |
+| `internal-public-blockchain`            | `internal-blockchain`  |
+| `internal-public-transaction-pool`      | `internal-transaction` |
+| `internal-public-tx-pool`               | `internal-tx-pool`     |
+| `internal-public-debug`                 | `internal-debug`       |
+| `internal-private-debug`                | `internal-debug`       |
+| `internal-public-account`               | `internal-account`     |
+| `internal-private-personal`             | `internal-personal`    |
+
+:::
+
+
 :::note
 
 If you populate this field, it will override the defaults so you must include every service you wish to enable.
 
 :::
 
-#### `eth`:
+#### `public-eth` (deprecated) or `eth`:
```

Comment:
@yulin-dong how do we normally handle deprecation notices like this?

It seems strange to keep the deprecated name in the header here. Should we move the old name and deprecation notice into the body and say something like:

```
The API name`public-eth` is deprecated as of v1.7.15, and the APIs previously under this name have been migrated to `eth`.
```

Reply:
Yeah, I like your suggestion better, @aaronbuchwald